app/src/main/java/org/wildaid/ofish/ui/search/simple/SimpleSearchViewModel.kt

```diff
@@ -50,7 +50,12 @@ class SimpleSearchViewModel(val repository: Repository, application: Application
 
         override fun initiateData() = dataSource
 
-        override fun applyFilter(filter: String) = dataSource
-            .filter { it.contains(filter, true) }
+        override fun applyFilter(filter: String): List<String> {
+            val result = dataSource.filter { it.contains(filter, true) }
+            if (result.isEmpty()) {
```

Comment:
We need to display other always . Not only when list is empty.  

And be sure that other will not be duplicate